js/script.js

```diff
@@ -0,0 +1,79 @@
+var runTheJewels = ["rtj3", "muthatruckas", "rain man didn't die face down", "talk real good", "cause im smart and stuff"]
+
+function lengths(rtj){
+	var wordLength = [];
+	for (i = 0; i < rtj.length; i++){
+		console.log(rtj[i].length)
+		wordLenght.push(rtj[i].length)
+	}
+	return wordLength
+}
+
+
+function transmogrifier(a, b, c){
+	var x = (a*b);
+	return Math.pow(x, c)
+}
+transmogrifier(5, 4, 3) = 8000
+transmogrifier(13, 12, 5) = 92389579776
+transmogrifier(42, 13, 7) = 14466001271480793000
+
+var daffy = 
+
+
+function toonify(accent, sentence){
+	if(accent === "daffy"){
+		return sentence.replace(/s/i, "th");
+	} else if (accent === "elmer"){
+		return sentence.replace(/r/i, "w");
+	} else {
+		return "please choose accent";
+	}
+}
+
+
+function wordReverse(stuff){
+	var arrayNew = stuff.split(" ");
+	arrayNew.reverse();
+	var retStr = "";
+	for (i = 0; i < arrayNew.length; i++){
+		retStr += arrayNew[i];
+		if(i == arrayNew.length-1){
+			console.log("Something went wrong");
+
```

Comment:
Nate, everything looks really good. Well written and everything works. This was the only line I could nit-pick because of the line-break, ha. Great job.